interfaces/builtin/fwupd_test.go

```diff
@@ -47,7 +48,7 @@ apps:
 const mockSlotSnapInfoYaml = `name: uefi-fw-tools
 version: 1.0
 apps:
- app:
+ app2:
```

Comment:
Just to have different security tags for plug and slot side, in the original tests both snap and app names on slot/plug side were the same. I guess I could change snap name.. Anyway the main point is to make it more apparent which security tag is expected.